controllers/api.template.js

```diff
@@ -2,11 +2,13 @@ const express = require('express');
 const model = require('../models/template-db');
 const createError = require('http-errors');
 const router = express.Router();
+// const url = require('url');
 
```

Comment:
Let's not leave unnecessary comments into the code - if something is not needed then remove it

Reply:
ok, i added it and then noticied it was not needed